src/app/parcels/form/ParcelForm.tsx

```diff
@@ -182,6 +213,21 @@ const ParcelForm: React.FC<ParcelFormProps> = ({
         <CenterComponent>
             <StyledForm>
                 <Title>Parcel Form</Title>
+                {clientDetails && (
+                    <StyledName>
+                        <h2>{clientDetails?.fullName}</h2>
```

Comment:
I think you can drop the `?` as we are checking `clientDetails` exists already

```suggestion
                        <h2>{clientDetails.fullName}</h2>
```